app/src/main/java/me/chester/minitruco/core/EstrategiaGasparotto.java

```diff
@@ -621,8 +620,8 @@ else if (carta.getValorTruco(s.manilha) == 14)
 		// utilizada diretamente
 		// Estas são cartas fictícias somente para os testes condicionais logo
 		// abaixo
-		Carta tres_testedemanilha = new Carta('3', 3);
-		Carta dois_testedemanilha = new Carta('2', 3);
+		Carta tres_testedemanilha = new Carta('3', Carta.Naipe.get(3));
+		Carta dois_testedemanilha = new Carta('2', Carta.Naipe.get(3));
```

Comment:
Maybe we could just say `Carta.Naipe.PAUS`?

(the original code should have used the constant anyway)